concepts/translations/base-language-selection.md

```diff
@@ -0,0 +1,49 @@
+# Defining Base Languages in Shopware’s Region-Independent Snippet Layer
+
+With Shopware 6.7 a **region-independent snippet layer** was introduced to reduce duplicate translations.  
```

Comment:
```suggestion
With Shopware 6.7 a **country-agnostic snippet layer** was introduced to reduce duplicate translations.  
```

In technical terms, we're always using "country-agnostic", so I'd prefer to do this everywhere